pkg/awsutil/sts.go

```diff
@@ -18,9 +18,9 @@ import (
 )
 
 const (
-	AwsFederatedSigninEndpoint = "https://signin.aws.amazon.com/federation"
-	AwsConsoleURL              = "https://console.aws.amazon.com/"
-	DefaultIssuer              = "Red Hat SRE"
+	AwsFederatedSigninEndpointTemplate = "https://%v.signin.aws.amazon.com/federation"
+	AwsConsoleURL                      = "https://console.aws.amazon.com/"
```

Comment:
This PR isn't adding these URLs, but why do these exist in backplane-cli now? These won't work in general for different partitions (e.g. GovCloud)

Reply:
Backplane API used to handle generating this url for clusters using the old flow. For clusters using isolated backplane flow, that isn't the case. So the CLI is responsible in those instances.